### PR TITLE
Add descriptive attribute semantics

### LINKED ISSUE #23: Add descriptive attribute semantics
* add semantic for `DescriptiveAttribute`
* add semantic for `ImpactAttribute `

### PR DESCRIPTION
**Pull request by bosyk**
closes #23

### CODE REVIEW

File: pylana/utils.py [hunk 1]
```diff
@@ -46,106 +68,110 @@ def create_semantics(columns: Iterable[str],
...
 
-    The columns semantic for lana will be derived from the data frame's dtypes. All types
-    will be converted to categorical attributes, besides numbers.
+    The Start and Complete time stamps need to have the same time format.
+    For an overview over time stamp formats see
+    https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#patterns
+
+    Args:
+        df:
+            A data frame representing amn event log.
```
Comment (Meissnerinc) [inline pylana/utils.py]: minor typo here "amn"

File: pylana/logs.py [hunk 2]
```diff
@@ -294,8 +318,12 @@ def upload_event_log_file(self, name: str,
             return self.post('/api/logs/csv-case-attributes-event-semantics',
                          files=files, data=semantics, **kwargs)
 
+    # TODO: Check whether impact and descriptive attributes are optional
```
Comment (Meissnerinc) [inline pylana/logs.py]: for the upload? yes, should be optional here to

**Review by Meissnerinc — COMMENTED** [review]
two minor comments, otherwise good to go

**Review by Meissnerinc — COMMENTED** [review]
two minor comments, otherwise good to go

**Review by Meissnerinc — APPROVED** [review]

File: pylana/logs.py (hunk 2 above)
Comment (bosyk) [reply pylana/logs.py]: I removed the TODO

**Pull request merged**